azext_iot/tests/iothub/jobs/test_iothub_jobs_int.py

```diff
@@ -182,6 +182,21 @@ def test_jobs(self):
             checks=[self.check("jobId", self.job_ids[2])],
         )
 
+        # Allow time for job to transfer to scheduled state (cannot cancel job in running state)
+        from time import sleep
+        sleep(5)
```

Comment:
Should we make this longer? Or is 5 second sleep consistently working.

Reply:
Consistently working for me, but I also haven't seen this issue pop up in the build pipelines, only in local testing - one of my hubs got stuck with a job pending in a `scheduled` state, and couldn't add a new job.

The error was because our test tried to cancel a job before it transitioned from `running` to `scheduled` and this seemed to fix it for me (at least for now)